db/index.js

```diff
@@ -1,5 +1,5 @@
 const Pool = require('pg').Pool;
-require('dotenv').config();
+//require('dotenv').config();
 
 const pool = new Pool({
   user:'brian.vu',
```

Comment:
Is this username going to mess things up w/ your version of postgres?

Reply:
lol i was super confused for a sec